lua/lspconfig/server_configurations/sonarlint.lua

```diff
@@ -0,0 +1,92 @@
+local util = require 'lspconfig.util'
+
+local tcp = vim.loop.new_tcp()
+tcp:bind('127.0.0.1', 0)
```

Comment:
You can't create a socket here. First of all it's going to be created by just configuring the server (even if it's never actually started), and second - now if there are multiple instances of this server, they will all try to use this socket.